.travis/rpmbuild_test

```diff
@@ -13,21 +13,20 @@ rpmbuild -ta *.tar.gz
 rpm -iv ~/rpmbuild/RPMS/*/*.rpm
 
 if [ "$OS_VERSION" = 6 ]; then
-    echo "the test suite has not yet been ported to centos6 python, skipping."
-    exit
+    echo "the python tests have not yet been ported to centos6 python, and will be skipped."
+else
```

Comment:
@dctrud did you know that python3 is available in EPEL6?  Just add the epel-release rpm and do 'yum install /usr/bin/python3'.

Reply:
Hi @DrDaveD - yep, am aware - but under standard installs on RHEL6 / CentOS6 we'd not be using this - so decided it wouldn't really add anything to have it tested against here. Cheers.

Reply:
I see, but FYI EPEL now requires that all packages capable of supporting python3 be compiled to use it, so when I installed singularity-2.5.1 in EPEL I enabled it.  So installs of singularity from EPEL will indeed use python3, and that's probably the place people using EL6 are most likely to get it. 

Anyway, thank you very much for enabling this on the EL6 travis build.  It's a big step up from no tests.  Running python3 tests additionally there are probably not a very big value because they the tests are also run on EL7.